sdk/identity/identity/src/index.ts

```diff
@@ -1,14 +1,41 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export * from "./plugins/consumer";
-
-export { IdentityPlugin } from "./plugins/provider";
-
 import { TokenCredential } from "@azure/core-auth";
+import { 
+  IdentityPlugin,
+  useIdentityPlugin,
+  MultiTenantTokenCredentialOptions,
+  AuthenticationRecord,
+  serializeAuthenticationRecord, 
+  deserializeAuthenticationRecord,
+  TokenCredentialOptions,
+  TokenCachePersistenceOptions,
+  logger,
+  AzureAuthorityHosts,
+  AuthenticationError,
+  ErrorResponse,
+  AggregateAuthenticationError,
+  AuthenticationErrorName,
+  AggregateAuthenticationErrorName,
+  CredentialUnavailableError,
+  CredentialUnavailableErrorName,
+  AuthenticationRequiredError,
+  AuthenticationRequiredErrorOptions } from "@azure/identity-common";
+
 import { DefaultAzureCredential } from "./credentials/defaultAzureCredential";
 
 export {
```

Comment:
can we do `export { /* things */ } from "@azure/identity-common";` instead of the import/export combo?